src/main/java/org/broadinstitute/hellbender/utils/test/SerializationTestUtils.java

```diff
@@ -0,0 +1,30 @@
+package org.broadinstitute.hellbender.utils.test;
+
+
+import org.apache.spark.SparkConf;
+import org.apache.spark.serializer.KryoSerializer;
+import org.apache.spark.serializer.SerializerInstance;
+import org.broadinstitute.hellbender.utils.Utils;
+import scala.reflect.ClassTag;
+import scala.reflect.ClassTag$;
+
+public final class SerializationTestUtils {
```

Comment:
You've created a duplicate class here -- this already exists in gatk public in `SparkTestUtils` in utils.test :)